### PR TITLE
some fixes for frontdoor readme

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: specification/frontdoor/resource-manager/readme.md [hunk 1]
```diff
@@ -28,17 +28,18 @@ These are the global settings for the FrontDoor API.
 title: FrontDoorManagementClient
 description: FrontDoor Client
 openapi-type: arm
-tag: package-2019-10
+tag: package-2019-11
```
Comment (yungezz) [inline specification/frontdoor/resource-manager/readme.md]: is `package-2019-10` released before?

Comment (zikalino) [reply specification/frontdoor/resource-manager/readme.md]: no, package-2019-10 and package-2019-11 were for some reason both created just several days ago.
it anything was released using one of these tags, package would be invalid.

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json [hunk 2]
```diff
@@ -726,7 +726,7 @@
               "Monthly"
             ],
             "x-ms-enum": {
-              "name": "aggregationInterval",
+              "name": "latencyScoreboardAggregationInterval",
```
Comment (yungezz) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: this is a potential breaking change, is this a swagger bug fixing to align with existing service behavior?

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json [hunk 3]
```diff
@@ -1110,16 +1112,16 @@
...
       "description": "Defines the server side resource status",
       "readOnly": true,
       "enum": [
-        "creating",
-        "enabling",
-        "enabled",
-        "disabling",
-        "disabled",
-        "deleting"
+        "Creating",
```
Comment (yungezz) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: again, breaking change

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 2 above)
Comment (minechosoft) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: this is fine :)

Comment (minechosoft) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: this should say "latencyScorecardAggregationInterval"

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json [hunk 4]
```diff
@@ -1044,7 +1046,7 @@
         },
         "resourceState": {
           "description": "Resource status.",
-          "$ref": "#/definitions/ResourceState"
+          "$ref": "#/definitions/NetworkExperimentationResourceState"
```
Comment (minechosoft) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: "NetworkExperimentResourceState"

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 3 above)
Comment (zikalino) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: discussed with service team, they will make sure RP is in sync (networkexperiment was not release yet)

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json [hunk 5]
```diff
@@ -868,14 +870,14 @@
       "properties": {
         "resourceState": {
           "description": "Resource status.",
-          "$ref": "#/definitions/ResourceState"
+          "$ref": "#/definitions/NetworkExperimentationResourceState"
```
Comment (minechosoft) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: "NetworkExperimentResourceState"

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 3 above)
Comment (minechosoft) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: yes. it's not breaking-- we will make sure to have these changes reflected in our service.

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 2 above)
Comment (zikalino) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: @minechosoft renamed to Scorecard

@yungezz changing name in **x-ms-enum** only results in SDK changes. anyway, we had to change this name, as there were two different enums with the same name, and that was causing autorest to fail. Any other changes I made were checked by @minechosoft and service is going to be aligned.

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 4 above)
Comment (zikalino) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: changed

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json (hunk 5 above)
Comment (zikalino) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-11-01/networkexperiment.json]: changed

**Review by yungezz — APPROVED** [review]

**Pull request merged**